app/code/Magento/MediaContent/Model/GetAssetIdByContentFieldComposite.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaContent\Model;
+
+use Magento\Framework\Exception\InvalidArgumentException;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\MediaContentApi\Api\GetAssetIdByContentFieldInterface as GetAssetIdByContentFieldApiInterface;
+use Magento\MediaContentApi\Model\GetAssetIdByContentFieldInterface;
+
+/**
+ * Class responsible to return Asset ids by content field
+ */
+class GetAssetIdByContentFieldComposite implements GetAssetIdByContentFieldApiInterface
+{
+    /**
+     * @var GetAssetIdByContentFieldInterface[]
+     */
+    private $getAssetIdByContentFieldArray;
+
+    /**
+     * GetAssetIdByContentStatusComposite constructor.
+     *
+     * @param array $getAssetIdByContentFieldArray
+     */
+    public function __construct($getAssetIdByContentFieldArray = [])
```

Comment:
I'd simplify the names to make the di configuration more compact and readable
```suggestion
    public function __construct($fieldHandlers = [])
```